web/public/stylesheets/wherehows.css

```diff
@@ -616,6 +616,7 @@ ul.breadcrumbs a {
 
 .search-feedback .btn {
   margin-left: 5px;
+
```

Comment:
Please remove this new line, you didn't change this file. This may cause further merge conflict.